### PR TITLE
minimal: fix format

### PR DESCRIPTION
**Pull request by zaoqi**

### CODE REVIEW

**Comment by paramat** [comment]
:+1:

**Comment by paramat** [comment]
It's not really worth opening a PR for 1 tiny formatting change when there are probably hundreds. However the change is fine.

File: games/minimal/mods/test/init.lua [hunk 1]
```diff
@@ -19,7 +19,7 @@ local function run_hpchangereason_tests(player)
 	assert(expect == nil)
 
 	expect = { a = 234, type = "set_hp", from = "mod" }
-	player:set_hp(7, { a= 234 })
+	player:set_hp(7, { a = 234 })
```
Comment (DS-Minetest) [inline games/minimal/mods/test/init.lua]: This line is touched, so you should correct all styling mistakes done here:
```suggestion
	player:set_hp(7, {a = 234})
```
(See https://dev.minetest.net/Lua_code_style_guidelines#Spaces.)

Comment (paramat) [reply games/minimal/mods/test/init.lua]: I agree.

**Comment by paramat** [comment]
This PR is too small and ignores many other format issues in the same file.
:-1:

**Pull request closed without merging**